contents/page-sections/authentication/with-image/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { Eye, EyeOff } from "@yamada-ui/lucide"
+import { Eye as EyeIcon, EyeOff as EyeOffIcon } from "@yamada-ui/lucide";
```

Comment:
`EyeIcon` and `EyeOffIcon` can also be imported directly.

```suggestion
import { EyeIcon, EyeOffIcon } from "@yamada-ui/lucide"
```